src/pages/settings.tsx

```diff
@@ -0,0 +1,39 @@
+import Layout from "../components/layout";
+import Button from "react-bootstrap/Button";
+import Image from "next/image";
+import Avatar from "../public/avatar.png";
+import { useRouter } from "next/router";
+
+const Settings = () => {
+  const router = useRouter();
+
+  const goBack = () => {
+    router.back();
+  };
+
+  return (
+    <Layout>
+      <div className="bg-dark d-flex flex-column vh-100 justify-content-center w-100">
```

Comment:
```suggestion
      <div className="bg-dark d-flex flex-column vh-100 justify-content-center align-items-center w-100">
```
-> dann kannst du die ganzen `mx-auto` weglassen